mlflow/langchain/__init__.py

```diff
@@ -929,7 +929,7 @@ def _patch_runnable_cls(cls):
     For classes that are subclasses of Runnable, we patch the `invoke`, `batch`, and `stream`
     methods for autologging.
     """
-    for func_name in ["invoke", "batch", "stream"]:
+    for func_name in ["invoke", "batch", "stream", "ainvoke", "abatch", "astream"]:
```

Comment:
`patched_inference` is non async function, right?

Reply:
Ah it's not, but it works 🤔

Reply:
I think what's happening is this:

```
import asyncio


async def original():
    await asyncio.sleep(1)
    return 0


def patch():
    return original()


async def main():
    print(await patch())


if __name__ == "__main__":
    asyncio.run(main())

```